buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -110,10 +140,10 @@ class ClusterFormationTasks {
      *
      * @return a task which starts the node.
      */
-    static Task configureNode(Project project, Task task, NodeInfo node) {
+    static Task configureNode(Project project, Task task, Object dependsOn, NodeInfo node, Configuration configuration) {
 
         // tasks are chained so their execution order is maintained
-        Task setup = project.tasks.create(name: taskName(task, node, 'clean'), type: Delete, dependsOn: task.dependsOn.collect()) {
+        Task setup = project.tasks.create(name: taskName(task, node, 'clean'), type: Delete, dependsOn: dependsOn) {
```

Comment:
you are not fully understanding the impact of this change apparently. for this entire cluster thing to work you have to share the `path.repo` since for the FS case it expects a shared filesystem. All nodes need to be able to access this. The only reason why this worked before was that we only had one node which had by definition access to this. The cleanup task per node was not necessary since it was implicitly done by unzipping the dist since it used to be under `${node.homeDir}`